specification/healthdataaiservices/HealthDataAIServices.Deidentification/tspconfig.yaml

```diff
@@ -0,0 +1,37 @@
+parameters:
+  "service-dir":
+    default: "sdk/deid"
```

Comment:
nit, folder and package name likely be wrong.

These can be modified after your SDK arch review.